src/Servers/Kestrel/shared/PooledStreamStack.cs

```diff
@@ -132,12 +139,12 @@ private static int CalculateRemoveCount(long now, int size, Http2StreamAsValueTy
             return size;
         }
 
-        internal readonly struct Http2StreamAsValueType
+        internal readonly struct StreamAsValueType
```

Comment:
Can we add this comment here? https://github.com/dotnet/runtime/blob/da9b16f2804e87c9c1ca9dcd9036e7b53e724f5d/src/libraries/System.IO.Pipelines/src/System/IO/Pipelines/BufferSegmentStack.cs#L68-L79